SQLitePCLRaw.lib.sqlite3mc/SQLitePCLRaw.lib.sqlite3mc.csproj

```diff
@@ -0,0 +1,44 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net8.0-android</TargetFrameworks>
+    <Description>Native binaries of the sqlite3mc library. Use SQLitePCLRaw.provider.sqlite3mc to access the library.</Description>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <IncludeBuildOutput>False</IncludeBuildOutput>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <AndroidNativeLibrary Include="..\cb\bld\bin\sqlite3mc\android\arm64-v8a\libsqlite3mc.so" />
+    <AndroidNativeLibrary Include="..\cb\bld\bin\sqlite3mc\android\armeabi-v7a\libsqlite3mc.so" />
+    <AndroidNativeLibrary Include="..\cb\bld\bin\sqlite3mc\android\x86\libsqlite3mc.so" />
+    <AndroidNativeLibrary Include="..\cb\bld\bin\sqlite3mc\android\x86_64\libsqlite3mc.so" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="buildTransitive\net8.0\SQLitePCLRaw.lib.sqlite3mc.targets" Pack="True" PackagePath="buildTransitive\net8.0" />
+    <None Include="lib\netstandard2.0\_._" Pack="True" PackagePath="lib\netstandard2.0" />
+    <None Include="..\cb\bld\bin\sqlite3mc\wasm\net8.0\sqlite3mc.a" Pack="True" PackagePath="runtimes\browser-wasm\nativeassets\net8.0" LinkBase="%(PackagePath)" />
```

Comment:
To keep things simple, I dropped support for a few things:

- .NET versions older than 8.0 (It's LTS and ships in November)
- Universal Windows Platform (But we should test that Packaged Windows App SDK projects still work)
- Xamarin (Users can target Android, iOS, etc. using .NET 8)

Reply:
I can't really judge whether supporting older .NET versions, UWP, and/or Xamarin is relevant or not. Nevertheless I'm in favor of keeping things simple. 😄 

In case there is a high demand for such support later on, it can be discussed again whether to add it or not.

Reply:
We should support .NET 6.0 as it's an LTS release and still something that is used quite widely. We should also support .net standard 2.0/2.1 if possible.